docs/en/contributor/Contributor-Tools.md

```diff
@@ -11,25 +11,9 @@ priority: 3
 
 ## IDE
 
-We recommend using either Eclipse or IntelliJ IDEA to contribute to Alluxio.
+We recommend using IntelliJ IDEA to contribute to Alluxio. Eclipse can also be used.
 Instructions for setting up both IDEs can be found below.
 
-### Eclipse
-
-You can generate an Eclipse configuration file by running:
-
-```console
-$ mvn clean -Pdeveloper install -DskipTests
-$ mvn clean -Pdeveloper -DskipTests eclipse:eclipse -DdownloadJavadocs=true -DdownloadSources=true
-```
-
-Then import the folder into Eclipse.
-You may also have to add the classpath variable `M2_REPO` by running:
-
-```console
-$ mvn -Declipse.workspace="your Eclipse Workspace" eclipse:configure-workspace
-```
-
 > Note: Alluxio 2.2 moved generated gRPC proto source files into `alluxio/core/transport/target/generated-sources/protobuf/`.
```

Comment:
this looks like an eclipse specific note that should also be moved accordingly?

Reply:
Yes, fixed.